File: Example/ExampleObjectiveC/EventViaDelegate+CoreDataProperties.m
//
//  EventViaDelegate+CoreDataProperties.m
//  ExampleObjectiveC
//
//  Created by zee-pk on 17/11/2016.
//  Copyright (c) 2016 predict.io by ParkTAG GmbH. All rights reserved.
//

#import "EventViaDelegate+CoreDataProperties.h"

@implementation EventViaDelegate (CoreDataProperties)

+ (NSFetchRequest<EventViaDelegate *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"EventViaDelegate"];
}

@dynamic accuracy;
@dynamic latitude;
@dynamic longitude;
@dynamic mode;
@dynamic timeStamp;
@dynamic type;
@dynamic stationary;
@dynamic zoneType;

@end

File: ExampleSwift/ExampleSwift/EventViaDelegate+CoreDataProperties.swift
//
//  EventViaDelegate+CoreDataProperties.swift
//  ExampleSwift
//
//  Created by Abdul Haseeb on 11/16/16.
//  Copyright © 2016 Abdul Haseeb. All rights reserved.
//

import Foundation
import CoreData


extension EventViaDelegate {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<EventViaDelegate> {
        return NSFetchRequest<EventViaDelegate>(entityName: "EventViaDelegate");
    }

    @NSManaged public var accuracy: NSNumber?
    @NSManaged public var latitude: NSNumber?
    @NSManaged public var longitude: NSNumber?
    @NSManaged public var mode: NSNumber?
    @NSManaged public var stationary: NSNumber?
    @NSManaged public var timeStamp: NSDate?
    @NSManaged public var type: NSNumber?
    @NSManaged public var zoneType: NSNumber?

}
